src/AbstractAlgebra.jl

```diff
@@ -18,15 +18,13 @@ import GroupsCore
 import GroupsCore: gens, ngens, order, mul!, istrivial
 
 # A list of all symbols external packages should not import from AbstractAlgebra
-import_exclude = [:import_exclude, :QQ, :ZZ,
+const import_exclude = [:import_exclude, :QQ, :ZZ,
                   :RealField, :number_field,
                   :AbstractAlgebra,
                   :inv, :log, :exp, :sqrt, :div, :divrem,
                   :numerator, :denominator,
                   :promote_rule,
                   :Set, :Module, :Group,
-                  # remove the following, once they land in OSCAR
-                  :allow_unicode, :with_unicode,
```

Comment:
@thofma I am not 100% sure I am reading this comment right: it seems we can remove these two lines here at some point ... and `allow_unicode`, `with_unicode` sure are in Oscar... But can we just remove this here, won't it lead to nasty warnings in Oscar about methods being redefined? Anyway, if that's the case, we could just mark this PR as "breaking", I guess?

Reply:
OK that makes more sense. Then we should first update Oscar accordingly. I've adjusted this PR to only update `with_unicode`